### PR TITLE
Switch to "Builder: oci-import"

### PR DESCRIPTION
**Pull request by tianon**
(https://github.com/docker-library/bashbrew/issues/51, https://github.com/docker-library/bashbrew/pull/61, https://github.com/docker-library/official-images/pull/13769, https://github.com/debuerreotype/debuerreotype/pull/149, https://github.com/docker-library/oi-janky-groovy/commit/e46741520620ebed245d6b2ea7bc0e10410fcfef :eyes:)

### CODE REVIEW

File: download.sh [hunk 1]
```diff
@@ -12,35 +12,39 @@ rm -v artifacts.zip
...
+		tar -xf "$dir/oci.tar" -C "$dir/oci"
+		rm -f "$dir/oci.tar" "$dir/rootfs.tar.xz"
+		rootfs='oci/blobs/rootfs.tar.gz'
+
+		[ -s "$dir/$rootfs" ]
+
+		cmd="$(jq -c '.config.Cmd' "$dir/oci/blobs/image-config.json")"
+		[[ "$cmd" = '["'*'"]' ]]
+
+		# this won't be used by us anymore, but included for backwards compatibility
```
Comment (AkihiroSuda) [inline download.sh:36]: Could you move this comment inside the Dockerfile for visibility?

Comment (tianon) [reply download.sh]: Great idea, updated!

**Review by myintn02 — APPROVED** [review]

**Review by yairk55 — APPROVED** [review]
nice

**Comment by tianon** [comment]
Almost exactly two years later, we can finally pull the trigger on this thanks to https://github.com/docker-library/meta-scripts/pull/93 :smile:

It's been a lot of work getting here, but it's nice to see the pieces all finally in place. :bow: :heart:

**Comment by tianon** [comment]
I have a rebuild planned for next week, so this will go into effect at that time :+1:

**Pull request merged**